src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java

```diff
@@ -130,10 +140,9 @@ public String version() {
     @Override
     public void open(Collection<TopicPartition> partitions) throws ConnectException {
         assignment.addAll(partitions);
-
         for (TopicPartition tp : assignment) {
             IngestionProperties ingestionProps = getIngestionProps(tp.topic());
-
+            log.warn("Open Kusto topic: " + tp.toString());
```

Comment:
warn? more like debug

Reply:
also probably be nicer to split partition and topic (makes it easier to parse values later)